### ISSUE TITLE
No effect on gameplay 

### ISSUE THREAD

**Issue by ZaneHD** [Soreepeong/XivAlexander#349]
So for whatever reason ever since i moved to OCE, my xivalexandwr has had no effect on my game and i was wondering if i needed to do something to make it work, i have already tried changing the latency modes aswell to no avail, it was working when i was on JP and also I've noticed my little window where I change everything has JP in the name of the window. Help please :) thanks in advance

**Comment by RealPeps** [comment]
Yeah I noticed that regardless of playing on NA or OCE it doesnt seem to have any effect since 6.08, this is without a 0ping vpn enabled AND with one enabled regardless of configuration.
It might be conflicting with XIVLauncher, which I've had issues with occasionally, and why I only do Load Once.

**Comment by ZaneHD** [comment]
> Yeah I noticed that regardless of playing on NA or OCE it doesnt seem to have any effect since 6.08, this is without a 0ping vpn enabled AND with one enabled regardless of configuration. It might be conflicting with XIVLauncher, which I've had issues with occasionally, and why I only do Load Once.

the probelm is i'm not even using XIVlauncher, and it's not working

**Comment by Oglaf** [comment]
Guys, the latest version is not updated to 6.08. Please check #331

**Comment by Oglaf** [comment]
> So for whatever reason ever since i moved to OCE, my xivalexandwr has had no effect on my game and i was wondering if i needed to do something to make it work, i have already tried changing the latency modes aswell to no avail, it was working when i was on JP and also I've noticed my little window where I change everything has JP in the name of the window. Help please :) thanks in advance

There is no Opcode for OCE server, you will have to get yourself. Check https://github.com/Soreepeong/XivAlexander/wiki/How-to-find-opcodes

**Comment by Soreepeong** [comment]
You can ignore that "JP" as it means that it's an international client. If you don't see any effects, try downloading it again, extract all, and then run from the new download. If it still doesn't work, try checking all tickboxes in Troubleshooting menu.

**Comment by robyn0751** [comment]
Issue popped up for me today, saw some "an invalid argument" errors in my log so is it related? I've tried reinstalling multiple times, have checked for updates as well
Does this version need updating after today's hotfixes?
[log.txt](https://github.com/Soreepeong/XivAlexander/files/8205632/log.txt)
